developer-docs/latest/getting-started/deployment.md

```diff
@@ -10,6 +10,38 @@ Deploying **databases** along with Strapi is covered in the [Databases Guide](..
 
 Manual guides for deployment on various platforms, for One-click and docker please see the [installation](../getting-started/installation.md) guides.
 
+<div>
+    <InstallLink link="../deployment/amazon-aws.html">
```

Comment:
This needs to change to `../deployment/21yunbox.html` and not the AWS Doc.

Reply:
updated.